src/sentry/utils/celery.py

```diff
@@ -1,9 +1,63 @@
+from collections.abc import Mapping, MutableSequence, Sequence
 from random import randint
 from typing import Any
 
 from celery.schedules import crontab
+from kombu import Queue
+
+from sentry.conf.types.celery import SplitQueueTaskRoute
 
 
 def crontab_with_minute_jitter(*args: Any, **kwargs: Any) -> crontab:
     kwargs["minute"] = randint(0, 59)
     return crontab(*args, **kwargs)
+
+
+def build_queue_names(base_name: str, quantity: int) -> Sequence[str]:
+    ret = []
+    for index in range(quantity):
+        name = f"{base_name}_{index + 1}"
+        ret.append(name)
+    return ret
+
+
+def _build_queues(base: str, quantity: int) -> Sequence[Queue]:
+    return [Queue(name=name, routing_key=name) for name in build_queue_names(base, quantity)]
+
+
+def make_split_task_queues(config: Mapping[str, SplitQueueTaskRoute]) -> Sequence[Queue]:
```

Comment:
Will we be using these methods to generate the queue list during application startup/config?

Reply:
We would use them to generate them in the settings file. 
The alternatives were:
- explicitly define each queues. I am pretty sure we would make mistakes there when duplicating them.
- During the Celery app initialization. That may work but I did not try that. I did not go that way because I am not confident that nobody uses CELERY_QUEUES before the Celery app is initialized. I think we should move this logic there after testing thoroughly it is ok.